valohai_yaml/pipelines/conversion.py

```diff
@@ -53,9 +77,13 @@ def convert_deployment_node(self, node: DeploymentNode) -> ConvertedObject:
         node_data["aliases"] = node.aliases
         return node_data
 
-    def convert_executionlike_node(self, node: Union[ExecutionNode, TaskNode]) -> ConvertedObject:
+    def convert_executionlike_node(
+            self, node: Union[ExecutionNode, TaskNode]
+    ) -> ConvertedObject:
         node_data = node.serialize()
-        node_data.pop('override', {})  # we'll use the actual object, not the serialization
+        node_data.pop(
+            "override", {}
+        )  # we'll use the actual object, not the serialization
```

Comment:
Is there an actual change hidden somewhere? I don't notice. Would be good to changes to format as a separated commit.

Reply:
these changes came from black command. I ran manually on my local.